src/vsr.zig

```diff
@@ -348,15 +354,13 @@ pub const Header = extern struct {
     fn invalid_ping(self: *const Header) ?[]const u8 {
         assert(self.command == .ping);
         if (self.parent != 0) return "parent != 0";
+        if (self.client != 0) return "client != 0";
         if (self.context != 0) return "context != 0";
         if (self.request != 0) return "request != 0";
+        if (self.view != 0) return "view != 0";
         if (self.commit != 0) return "commit != 0";
         if (self.timestamp != 0) return "timestamp != 0";
-        if (self.view != 0) return "view != 0";
-        if (self.client != 0) {
-            if (self.replica != 0) return "replica != 0";
-            if (self.op != 0) return "op != 0";
-        }
+        if (self.size != @sizeOf(Header)) return "size != @sizeOf(Header)";
```

Comment:
:+1:

Reply:
Thinking about this ... if we assert that we have no body, shouldn't we also assert about `checksum_body`?